packages/astro/src/runtime/server/render/any.ts

```diff
@@ -3,52 +3,134 @@ import { isPromise } from '../util.js';
 import { isAstroComponentInstance, isRenderTemplateResult } from './astro/index.js';
 import { type RenderDestination, isRenderInstance } from './common.js';
 import { SlotString } from './slot.js';
-import { renderToBufferDestination } from './util.js';
+import { createBufferedRenderer } from './util.js';
 
-export async function renderChild(destination: RenderDestination, child: any) {
+export function renderChild(destination: RenderDestination, child: any): void | Promise<void> {
```

Comment:
I always thought this was one of the big bottlenecks. Thank you so much for doing this!